backend/entity/user.go

```diff
@@ -1,8 +1,94 @@
-package entity
+package main
```

Comment:
there is data here that only is related to the candidate (not the user in general) - we need to find a way to parse it accordingly